cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -608,7 +616,7 @@ reportBuildFailures verbosity plan buildOutcomes
 
        -- For all failures, print either a short summary (if we showed the
        -- build log) or all details
-       die $ unlines
+       notice verbosity $ unlines
```

Comment:
You're quite right, it is. Do we think this is a real problem? If a user uses -v0 do they expect this kind of output? Shouldn't it really mean no output (even for die messages)?